polaris-react/src/components/ResourceList/ResourceList.stories.tsx

```diff
@@ -290,17 +293,26 @@ export function WithBulkActionsAndManyItems() {
   ];
 
   return (
-    <Card padding="0" roundedAbove="sm">
-      <ResourceList
-        resourceName={resourceName}
-        items={items}
-        renderItem={renderItem}
-        selectedItems={selectedItems}
-        onSelectionChange={setSelectedItems}
-        promotedBulkActions={promotedBulkActions}
-        bulkActions={bulkActions}
-      />
-    </Card>
+    <BlockStack gap="400">
+      <Card padding="0" roundedAbove="sm">
```

Comment:
```suggestion
      <Card padding="0">
```

Card was recently updated to set `roundedAbove` to `sm` by default.